openmdao/core/system.py

```diff
@@ -191,10 +191,14 @@ class System(object):
         If True, this system approximated its Jacobian
     _owns_approx_jac_meta : dict
         Stores approximation metadata (e.g., step_size) from calls to approx_total_derivs
-    _owns_approx_wrt : set or None
-        Overrides aproximation inputs.
     _owns_approx_of : set or None
         Overrides aproximation outputs.
+    _owns_approx_of_idx : dict
```

Comment:
these descriptions are a little vague. Is the override only happening in the context of FD? Could you make them a bit more clear?